.travis.yml

```diff
@@ -6,10 +6,13 @@ language: python
 python:
   - "2.7"
 
+addons:
+  postgresql: "9.4"
+
 cache:
   directories:
     - $HOME/.cache/pip
-    - $HOME/caffe
+    #- $HOME/caffe
```

Comment:
We need one PR merged with this line commented to update the Caffe version in master branch.